app/src/test/java/org/simple/clinic/home/patients/PatientsScreenControllerTest.kt

```diff
@@ -177,6 +159,8 @@ class PatientsScreenControllerTest {
     whenever(userSession.loggedInUser()).thenReturn(Observable.just(Just(user)))
     whenever(hasUserDismissedApprovedStatus.asObservable()).thenReturn(Observable.just(false))
     whenever(hasUserDismissedApprovedStatus.get()).thenReturn(false)
+    whenever(userSession.refreshLoggedInUser()).thenReturn(Completable.never())
```

Comment:
We could move this to the `setUp` fixture and only override it in the tests that depend on this?